### PR TITLE
Update lunch me script to support current Yelp API

### PR DESCRIPTION
**Pull request by erinknight242**
Includes searches for Monterrey, and a help blurb

### CODE REVIEW

File: scripts/lunchme.coffee [hunk 1]
```diff
@@ -6,14 +6,24 @@
 #   hubot lunch me tacos - find what you are in the mood for
 #   hubot houston lunch me - find a place in Houston, too!
 
-radius = 5
+yelp = require "yelp"
+
+# Generate new keys at https://www.yelp.com/developers/manage_api_keys
+yelp = yelp.createClient
+  consumer_key: "uh4_dL5KECQPgl5OlS9AVg"
```
Comment (hulahomer) [inline scripts/lunchme.coffee]: I know we had the yelp api key here in plain view before, but what are your thoughts on pushing these 4 values into env variables that we can include in the bash script that sets them on the server?

Comment (erinknight242) [inline scripts/lunchme.coffee]: That would be cool. Do we have another script doing that for me to look at?

Comment (hulahomer) [inline scripts/lunchme.coffee]: We sure do. I'll send you the details

**Comment by erinknight242** [comment]
this commit fixes #62

**Comment by erinknight242** [comment]
also fixes #6 and #4

**Pull request merged**